extensions/ql-vscode/src/languageSupport.ts

```diff
@@ -17,11 +17,14 @@ export function install() {
   // setLanguageConfiguration requires a regexp for the wordpattern, not a string
   langConfig.wordPattern = new RegExp(langConfig.wordPattern);
   langConfig.onEnterRules = onEnterRules;
+  langConfig.indentationRules = {
+		increaseIndentPattern: /^((?!.*?\/\*).*\*\/)?\s*[\}\]].*$/,
+		decreaseIndentPattern: /^((?!\/\/).)*(\{[^}"'`]*|\([^)"'`]*|\[[^\]"'`]*)$/
+	};
```

Comment:
This still looks to me like these are back to front. You seem to have `}` and `]` increasing the indentation and `{` and [`  increasing it.

Reply:
These are the same patterns suggested by https://code.visualstudio.com/api/language-extensions/language-configuration-guide so as long as they are swapped as suggested this lgtm.